step-api/pom.xml

```diff
@@ -1,65 +1,38 @@
-<!--
```

Comment:
I realize the copyright has been removed from most parent pom.xml (also in other refactored repo), I would had them back